### PR TITLE
WSDEV-860 fix style for event and video variant

### PR DESCRIPTION
**Pull request by HanqingZhou**
Event variant:
before:
Screen Shot 2022-03-21 at 9 08 57 AM

after: 

Screen Shot 2022-03-21 at 9 47 44 AM

Video variant:
before:
Screen Shot 2022-03-21 at 8 56 04 AM

after:  
Screen Shot 2022-03-21 at 8 57 04 AM

### CODE REVIEW

File: src/blocks/Listing/Listing.scss [hunk 1]
```diff
@@ -50,7 +50,6 @@
 
     header,
     div {
-        flex: 1;
```
Comment (marceloglacial) [inline src/blocks/Listing/Listing.scss:53]: Good job!

File: src/blocks/Listing/components/ListingVideo.tsx [hunk 2]
```diff
@@ -35,7 +35,7 @@ const ListingVideo: React.FC<
                                     />
                                     <svg
                                         aria-hidden='true'
-                                        viewBox='0 0 24 24'
```
Comment (marceloglacial) [inline src/blocks/Listing/components/ListingVideo.tsx:38]: Please add this icon to the icon library :)

**Review by marceloglacial — CHANGES_REQUESTED** [review]

File: src/blocks/Listing/Listing.scss (hunk 1 above)
Comment (HanqingZhou) [reply src/blocks/Listing/Listing.scss:53]: Thank you

File: src/blocks/Listing/components/ListingVideo.tsx (hunk 2 above)
Comment (HanqingZhou) [reply src/blocks/Listing/components/ListingVideo.tsx:38]: Done, added to Icon library

**Review by marceloglacial — APPROVED** [review]
Grat job!

**Pull request merged**